R/algo-lmfx.R

```diff
@@ -0,0 +1,92 @@
+#' Individual Tree Detection Algorithm
+#'
+#' Copied from: https://raw.githubusercontent.com/Jean-Romain/lidRplugins/c42cf582ca28b315bf4166f88d727cd80866fafa/R/algo_tree_detection.r
```

Comment:
Remove this line

Reply:
done! Just wanted to make sure credit goes to the right person